_jobs/python-software-developer.md

```diff
@@ -0,0 +1,95 @@
+---
+product-title: Python Software Developer
+product-description: The Python software developer will join our DevOps team using Python and the latest container technology (Docker and Kubernetes), to further develop the NuvlaEdge software, which turns any edge device into a smart and connected device, managed by the Nuvla.io service.
+vignette: python-software-developer-vignette.png
+---
+
+The Python software developer will join the dev team and contribute to the NuvlaEdge software solution. Able to run on any ARM or x86 hardware architectures, the NuvlaEdge software turns any edge device into a smart and connected device, managed by the Nuvla.io service. Using Docker and Kubernetes, this highly available software is at the core of SixSq quest to disrupt the IT industry.
+
+### Description
+
+SixSq is building the most advanced edge-to-cloud B2B SaaS platform: Nuvla.io.  With the recent acquisition of SixSq by Ekinops, we are growing. We are looking for software developers, wanting to join a multi-cultural and international team.  As a software developer you will contribute to the overall objective of the company, which is to disrupt the IT industry with an open, agile, simple and efficient platform. The platform also includes a marketplace of business applications targeting the far edge in a range of sectors such as retail, manufacturing, transport, telecoms and energy.  In time, since we sell as a team, you will also be asked to support sales efforts via creative demonstrations and technical discussions with customers and prospects.
+
+To communicate with Nuvla.io, edge devices run the NuvlaEdge software.  Packaged as micro-services (e.g. Docker and Kubernetes), NuvlaEdge provides the local environment that allows Nuvla.io to manage fleets of edge devices, as well as the life-cycle of business apps running at the edge.
```

Comment:
"Packaged as micro-services (e.g. Docker and Kubernetes)," -> "Designed and implemented as microservices and running on Docker and Kubernetes,"